<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>1097</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>209400234-杨钧升</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>1000</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>1000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>3</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="background">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="picLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Dock"/>
    <addaction name="action_S"/>
    <addaction name="action_Save"/>
   </widget>
   <widget class="QMenu" name="menu_F_2">
    <property name="title">
     <string>翻译(&amp;F)</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionChinese"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_F_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_S"/>
   <addaction name="action_Save"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>38</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>工具箱</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>字符画</string>
     </property>
    </widget>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>2</number>
     </property>
     <property name="placeholderText">
      <string>字体大小</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>1</number>
     </property>
     <property name="placeholderText">
      <string>显示字符</string>
     </property>
    </widget>
    <widget class="QPushButton" name="grayBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>灰度化(&amp;R)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="filteringBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>均值滤波(&amp;T)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>保存(&amp;S)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="watermarkBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>水印(&amp;I)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="brightnessBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>亮度(&amp;Y)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edgeBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>边缘检测(&amp;U)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/image/images/filenew.png</normaloff>:/image/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>新建文件(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Dock">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/image/images/find.png</normaloff>:/image/images/find.png</iconset>
   </property>
   <property name="text">
    <string>显示Dock(&amp;D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_S">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/image/images/zoom-in.png</normaloff>:/image/images/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>图片选择(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>图片选择(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="myimages.qrc">
     <normaloff>:/image/images/filesave.png</normaloff>:/image/images/filesave.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionChinese">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myimages.qrc"/>
 </resources>
 <connections/>
</ui>
